builders/xcm/xcm-sdk/reference.md

```diff
@@ -0,0 +1,65 @@
+---
+title: XCM SDK Reference
+description: A reference for the interfaces and methods in the Moonbeam XCM SDK, which can be used to send XCM transfers between Moonbeam and other chains in the ecosystem.
+---
+
+# Moonbeam XCM SDK Reference
+
+![XCM SDK Banner](/images/builders/xcm/sdk/reference-banner.png)
+
+## Introduction {: #introduction }
+
+The Moonbeam XCM SDK enables developers to easily deposit and withdraw assets to Moonbeam/Moonriver from the relay chain and other parachains in the Polkadot/Kusama ecosystem. With the SDK, you don't need to worry about determining the multilocation of the origin or destination assets or which extrinsics are used on which networks to send XCM transfers.
+
+The SDK provides an API which includes a series of interfaces to get asset information for each of the supported assets, chain information for the initialized network, utility methods, and methods to enable deposits, withdrawals, and subscription to balance information.
+
+This page includes a list of the interfaces and methods available in the XCM SDK. For information on how to use the XCM SDK interfaces and methods, please refer to the [Using the XCM SDK](/builders/xcm/xcm-sdk/xcm-sdk){target=_blank} guide.
+
+## Core Interfaces {: #core-sdk-interfaces }
+
+The SDK provides the following core interfaces, which can be accessed after [initialization](/builders/xcm/xcm-sdk/xcm-sdk/#initializing):
+
+|                                   Interface                                    |                                                                         Description                                                                         |
+|:------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|       [`symbols`](/builders/xcm/xcm-sdk/xcm-sdk/#symbols){target=_blank}       |                   A list containing the asset's origin chain symbol for each of the supported assets for the initialized Moonbeam network                   |
+|        [`assets`](/builders/xcm/xcm-sdk/xcm-sdk/#assets){target=_blank}        |    A list of the supported assets for the initialized Moonbeam network along with their asset ID, precompiled address on Moonbeam, and the asset symbol     |
+|   [`moonAsset`](/builders/xcm/xcm-sdk/xcm-sdk/#native-assets){target=_blank}   |                      Contains the asset ID, precompile contract address, and native asset symbol for the initialized Moonbeam network                       |
+| [`moonChain`](/builders/xcm/xcm-sdk/xcm-sdk/#native-chain-data){target=_blank} | Contains the chain key, name, WSS endpoint, parachain ID, decimals of the native asset, chain ID, and units per second for the initialized Moonbeam network |
+
+## Core Methods {: #core-sdk-methods }
+
+The SDK provides the following core methods, which will be covered in this guide:
```

Comment:
I'm guessing that you say `in this guide` because this was in the xcm SDK guide before?